bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -74,8 +69,46 @@ local_setup() {
     try --max 48 --delay 5 running_nginx
 }
 
+@test 'verify identification errors' {
+    run rdctl snapshot restore 'the-nomadic-pond'
+    assert_failure
+    assert_output --partial $"Error: can't restore snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot restore 'the-nomadic-pond' --json
+    assert_failure
+    run jq_output '.error'
+    assert_success
+    assert_output --partial $"can't restore snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot delete 'the-nomadic-pond'
+    assert_failure
+    assert_output --partial $"Error: can't delete snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+    run rdctl snapshot delete 'the-nomadic-pond' --json
+    assert_failure
+    run jq_output '.error'
+    assert_success
+    assert_output $"can't delete snapshot: can't find a snapshot with name \"the-nomadic-pond\""
+}
+
+@test 'can create a snapshot where proposed name is a current ID' {
```

Comment:
It would be better if this were a unit test. Though I'm not sure how much it matters, given that we want to release.